drive-characterization/data_logger.py

```diff
@@ -215,6 +215,11 @@ def run(self):
         else:
             NetworkTables.startClientTeam(int(team))
 
+        # Make the user pick linear or angular mode
+        angular_mode = input("Choose 'linear' or 'angular' mode: ") == "angular"
+        if angular_mode:
+            print("You have chosen angular mode; you must invert one of your robot's motors for this to work correctly!")
```

Comment:
I'm confused, why is this warning there?

Reply:
If the user hasn’t read the documentation then they might assume the robot will just spin in place, even if they haven’t properly set their motor directions.

We can remove this, but it seems like it might not be obvious that you have to do this manually.